src/Core/Utilities/Fido2/Fido2SelectedCredential.cs

```diff
@@ -0,0 +1,10 @@
+using Bit.Core.Models.View;
+
+public class Fido2SelectedCredential
+{
+    public byte[] Id { get; set; }
+
+    public byte[] UserHandle { get; set; }
+
+    public CipherView Cipher { get; set; }
```

Comment:
🤔 I see that before this was being marked as `nullable`, is that not the case anymore?:
```suggestion
    #nullable enable
    public byte[]? UserHandle { get; set; }

    public CipherView? Cipher { get; set; }
```

Reply:
Sort of, according to the spec this is optional in the response, but required in the request. We always save it from the request, and so we have no reason to allow this to be nullable in the response. 

> Its value’s [name](https://www.w3.org/TR/webauthn-3/#dom-publickeycredentialentity-name), [displayName](https://www.w3.org/TR/webauthn-3/#dom-publickeycredentialuserentity-displayname) and [id](https://www.w3.org/TR/webauthn-3/#dom-publickeycredentialuserentity-id) members are REQUIRED. [id](https://www.w3.org/TR/webauthn-3/#dom-publickeycredentialuserentity-id) can be returned as the [userHandle](https://www.w3.org/TR/webauthn-3/#dom-authenticatorassertionresponse-userhandle) in some future [authentication ceremonies](https://www.w3.org/TR/webauthn-3/#authentication-ceremony),
> -- https://www.w3.org/TR/webauthn-3/#dictionary-makecredentialoptions